packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-keys.js

```diff
@@ -0,0 +1,41 @@
+const loginEnBundle = require('../../../i18n/src/json/login.json');
+const loginokPLBundle = require('../../../i18n/src/json/login_ok_PL.json');
+const _ =  require('underscore');
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Detect missing keys from login_ok_PL.properties',
+      category: 'i18n Issues',
+      recommended: true,
+    },
+    messages: {
+      noMissingKeys: 'login.properties ( {{noOfKeysLoginEn}}: keys ) and login_ok_PL.properties ( {{noOfKeysLoginokPL}}: keys), are out of sync. Missing keys from login_ok_PL.properties. Run grunt exec:pseudo-loc to re-generate login_ok_PL.properties.'
```

Comment:
I suggest cutting down on this error message length. We can omit some of the content here as it should be implied:

```suggestion
      noMissingKeys: 'login_ok_PL.properties is not up-to-date with login.properties. Please run "grunt exec:pseudo-loc" to synchronize.'
```